FitGoal/FitGoal/GoalTrakerCell.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  GoalTrakerCell.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 8/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class GoalTrakerCell: UICollectionViewCell {
+    static var identifier = "GoalTraker"
+    
+    lazy var topView: UIView = {
```

Comment:
What do you think about using a different name for this view? It isn't a top view anymore, it looks more a background.